torch/_decomp/decompositions.py

```diff
@@ -1136,6 +1136,19 @@ def addmm(self: Tensor, mat1: Tensor, mat2: Tensor, beta: int = 1, alpha: int =
     return out + beta * self
 
 
+@register_decomposition(aten.addmv)
+@out_wrapper()
+@pw_cast_for_opmath
+def addmv(self: Tensor, mat1: Tensor, vec: Tensor, beta=1, alpha=1):
+    if not self.is_floating_point() and not self.is_complex():
+        beta = int(beta)
+        alpha = int(alpha)
+    out = alpha * torch.matmul(mat1, vec)
```

Comment:
I think we should decompose it to `torch.mv`, as we are doing with `torch.addmm`. I wouldn't worry about some minor accuracy issues, which may very well be coming from the use of FMA, as Natalia and Ed described.

About whether to add an FMA prim or not, it may actually make sense. One of the design principles for PrimTorch was that anything that had its own function on the STL would be a prim. This makes sense, as the STL chooses these considering their speed / numerical stability. In this case, it looks like FMA may fit both speed and stability so well worth having it IMO. Perhaps, in this case, this could be not a prim but a decomposition, and then backends could choose to override it with their own implementation?